packages/@react-spectrum/datepicker/stories/DateRangePicker.stories.tsx

```diff
@@ -89,10 +89,19 @@ storiesOf('Date and Time/DateRangePicker', module)
   )
   .add(
     'isDateUnavailable',
-    () => render({isDateUnavailable: (date: DateValue) => {
-      const disabledIntervals = [[today(getLocalTimeZone()), today(getLocalTimeZone()).add({weeks: 1})], [today(getLocalTimeZone()).add({weeks: 2}), today(getLocalTimeZone()).add({weeks: 3})]];
-      return disabledIntervals.some((interval) => date.compare(interval[0]) > 0 && date.compare(interval[1]) < 0);
-    }})
+    () => {
```

Comment:
Some feedback on the VO experience with this range calendar:: I found it hard to tell which dates specifically were invalid in the date range. Every date is announced as "invalid data" but the truly invalid dates were the "dimmed invalid data" ones. Would we expect the end user to provide an error message that specifies the invalid dates or could we add additional labeling on the dates themselves to better indicate what dates are breaking validation? Maybe a label on the first date in the range that says "invalid dates: may 5th, 6th, etc"? This might be hard/unfeasible if there were a lot of invalid date (very verbose) and it might not be possible to get a list of the unavailable dates (I think only visible dates evaluate if they are unavailable right? Might be remembering incorrectly)

Reply:
Yeah, we don't have that information. In this particular example it might work, but applications can do any sort of validation they want. The selected range is what's invalid, not necessarily any specific dates. I think it will be on applications to provide a good error message.